db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170609214859) do
+ActiveRecord::Schema.define(version: 20170619213358) do
 
```

Comment:
I'll clean up the DB renames. The open schema should be cleaned up so it matches a db:migrate from scratch, but I often run off of an older database. I should probably figure out if I can copy all my data over to a fresh schema...

Can you also take another look over #1111 as this depends on that.